types/node/v6/base.d.ts

```diff
@@ -1685,6 +1685,7 @@ declare module "vm" {
         columnOffset?: number;
         displayErrors?: boolean;
         timeout?: number;
+        breakOnSigint?: boolean;
```

Comment:
Please do not update any unsupported nodejs versions, only LTS and ACTIVE.

Reply:
Thanks you for your comment. Reverted changes on v6, 7, 8, 9 and 11.